src/components/calculator.js

```diff
@@ -0,0 +1,157 @@
+        </td>
+        <td>
+          <button
+            type="button"
+            className="calculator__key calculator__key--operator"
+            value="%"
+          >
+            %
+          </button>
+        </td>
+        <td>
+          <button
+            type="button"
+            className="calculator__key calculator__key--operator"
+            value="/"
+          >
+            &divide;
+          </button>
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <button type="button" className="calculator__key" value="7">
+            7
+          </button>
+        </td>
+        <td>
+          <button type="button" className="calculator__key" value="8">
+            8
+          </button>
+        </td>
+        <td>
+          <button type="button" className="calculator__key" value="9">
+            9
+          </button>
+        </td>
+        <td>
+          <button
+            type="button"
+            className="calculator__key calculator__key--operator"
+            value="*"
+          >
+            &times;
+          </button>
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <button type="button" className="calculator__key" value="4">
+            4
+          </button>
+        </td>
+        <td>
+          <button type="button" className="calculator__key" value="5">
+            5
+          </button>
+        </td>
+        <td>
+          <button type="button" className="calculator__key" value="6">
+            6
+          </button>
+        </td>
+        <td>
+          <button
+            type="button"
+            className="calculator__key calculator__key--operator"
+            value="-"
+          >
+            -
+          </button>
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <button type="button" className="calculator__key" value="1">
+            1
+          </button>
+        </td>
+        <td>
+          <button type="button" className="calculator__key" value="2">
+            2
+          </button>
+        </td>
+        <td>
+          <button type="button" className="calculator__key" value="3">
+            3
+          </button>
+        </td>
+        <td>
+          <button
+            type="button"
+            className="calculator__key calculator__key--operator"
+            value="+"
+          >
+            +
+          </button>
+        </td>
+      </tr>
+      <tr>
+        <td colSpan={2}>
+          <button type="button" className="calculator__key zero" value="0">
+            0
+          </button>
+        </td>
+
+        <td>
+          <button type="button" className="calculator__key" value=".">
+            .
+          </button>
+        </td>
+        <td>
+          <button
+            type="button"
+            className="calculator__key calculator__key--operator"
+            value="="
+          >
+            =
+          </button>
+        </td>
+      </tr>
+    </table>
+  </div>
+);
+
```

Comment:
- [ ] kindly put this in a separate component and import it here, this is good for reusability and makes your code base more professional

Reply:
Yeah good idea @VICTOR-CHUKWU